extensions/amp-form/0.1/amp-form.js

```diff
@@ -209,8 +206,9 @@ export class AmpForm {
   getXhrUrl_(attribute) {
     const url = this.form_.getAttribute(attribute);
     if (url) {
-      assertHttpsUrl(url, this.form_, attribute);
-      user().assert(!isProxyOrigin(url),
+      const urlService = Services.urlForDoc(this.element);
```

Comment:
`AmpForm` doesn't inherit from `AMP.BaseElement` so it doesn't have a `this.element`. These need to be `this.form_`.

Reply:
Yup, fixed.